moderncvheadv.sty

```diff
@@ -91,23 +93,23 @@
       % name and optional title
       \newlength{\makecvheadpictureboxskip}%
       \setlength{\makecvheadpictureboxskip}{\totalheightof{\usebox{\makecvheadpicturebox}}}%
-      \namestyle{\@firstname\ \@lastname}%
+      \firstnamestyle{\@firstname}~\lastnmestyle{\@lastname}%
```

Comment:
Should this not be `\lastnamestyle`? Maybe a typo?

Reply:
solved with [7ea5018](https://github.com/moderncv/moderncv/pull/151/commits/7ea50184fc2e6acc76de781a0c8fcf8f67cd7c28)